egs/ami/s5/local/tfrnnlm/run_lstm.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+mic=ihm
+ngram_order=4
+stage=1
+weight=0.5
+
+. ./utils/parse_options.sh
+. ./cmd.sh
+. ./path.sh
+
+set -e
+
+dir=data/tensorflow_lstm
+mkdir -p $dir
+
+if [ $stage -le 1 ]; then
+  local/tfrnnlm/rnnlm_data_prep.sh $dir
+fi
+
+mkdir -p $dir
+if [ $stage -le 2 ]; then
+# the following script uses TensorFlow. You could use tools/extras/install_tensorflow_py.sh to install it
```

Comment:
This script doesn't check if TensorFlow is installed. If it's not installed it should die with a message explaining how to install it.   This script should also check whether the TensorFlow-related binaries have been compiled, and if not, explain how to compile them.